internal/generator/binding/annotation.go

```diff
@@ -32,6 +32,27 @@ type Annotation struct {
 	Details map[string]*Annotation
 }
 
+func (a Annotation) HasDetail(name string) bool {
+	return a.Details != nil && a.Details[name] != nil
+}
+
+func HasBooleanDetail(annotations map[string]*Annotation, parentAnnotation, name string) (bool, error) {
+	if annotations[parentAnnotation] != nil {
+		a := annotations[parentAnnotation]
+		if a.Details != nil && a.Details[name] != nil {
```

Comment:
Was `HasDetail(string)` supposed to replace this? Otherwise it appears unused.

Reply:
cleaned up